CI/integration_tests/calculators/test_green_kubo_ionic_conductivity.py

```diff
@@ -65,4 +66,12 @@ def test_project(traj_file, true_values, tmp_path):
 
     computation = project.run.GreenKuboIonicConductivity(plot=False)
 
+    # Time is wrong in the test data
+    computation["NaCl"]["System"].pop("time")
+    computation["NaCl"]["System"].pop("integral")
+    computation["NaCl"]["System"].pop("integral_uncertainty")
+
+    true_values["System"].pop("time")
+    true_values["System"]["acf"] = (np.array(true_values["System"]["acf"]) / 500).tolist()
```

Comment:
What's that 500?

Reply:
There was a multiply by data range and then what followed after the saving was a divide by data range so there was no effect, I got rid of them both but the data saved on ZnHub had the redundant prefacto which in this specific case is 500. I will remove the online data for the test and try to find a best analytical solution

Reply:
Perfect.